src/scripts/main.js

```diff
@@ -1,3 +1,332 @@
+//     return;
+//   }
+
+//   if (!position) {
+//     showNotification('Error', 'Please choose a valid position.', 'error');
+
+//     return;
+//   }
+
+//   if (age < 18 || age > 90) {
+//     showNotification('Error', 'Age must be between 18 and 90.', 'error');
+
+//     return;
+//   }
+
+//   if (isNaN(salary) || salary <= 0) {
+//     showNotification('Error', 'Salary must be a positive number.', 'error');
+
+//     return;
+//   }
+
+//   addEmployeeToTable(nameEmployee, position, office, age, salary);
+//   showNotification('Success', 'Employee added successfully!', 'success');
+// }
+
+function validateForm() {
+  const nameEmployee = form.elements.name.value;
+  const position = form.elements.position.value;
+  const office = form.elements.office.value;
+  const age = parseInt(form.elements.age.value, 10);
+  const salary = parseFloat(form.elements.salary.value);
+
+  const validationError = validateEmployeeDetails(
+    nameEmployee,
+    position,
+    age,
+    salary,
+  );
+
+  if (validationError) {
+    showNotification('Error', validationError, 'error');
+
+    return;
+  }
+
+  addEmployeeToTable(nameEmployee, position, office, age, salary);
+  showNotification('Success', 'Employee added successfully!', 'success');
+}
+
+function validateEmployeeDetails(nameWorker, position, age, salary) {
+  if (nameWorker.length < 4) {
+    return 'Name must be at least 4 characters long.';
+  }
+
+  if (!position) {
+    return 'Please choose a valid position.';
+  }
+
+  if (age < 18 || age > 90) {
+    return 'Age must be between 18 and 90.';
+  }
+
+  if (isNaN(salary) || salary <= 0) {
+    return 'Salary must be a positive number.';
+  }
+
+  return null;
+}
+
+// function showNotification(title, message, type) {
+//   const notification = document.createElement('div');
+
+//   notification.classList.add('notification');
+//   notification.classList.add(type);
+//   notification.setAttribute('data-qa', 'notification');
+
+//   const titleElement = document.createElement('span');
+
+//   titleElement.classList.add('title');
+//   titleElement.textContent = title;
+
+//   const messageElement = document.createElement('span');
+
+//   messageElement.textContent = message;
+
+//   notification.appendChild(titleElement);
+//   notification.appendChild(messageElement);
+//   document.body.appendChild(notification);
+
+//   setTimeout(() => {
+//     notification.style.visibility = 'hidden';
+//   }, 1000);
+// }
+
+function showNotification(title, message, type) {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+  notification.setAttribute('data-qa', 'notification');
+
+  notification.innerHTML = `
+    <span class="title">${title}</span>
+    <span>${message}</span>
+  `;
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
+  }, 1000);
+}
+
+function addEmployeeToTable(nameEmployee, position, office, age, salary) {
```

Comment:
The function `addEmployeeToTable` is doing more than adding an employee to the table. It's also resetting the form. Consider renaming this function to reflect this, or better yet, separate the concerns of adding an employee and resetting the form into different functions.